reorganize.py

```diff
@@ -59,11 +59,13 @@ def init_logger(level):
 if __name__ == '__main__':
     source, destination, extensions, debug = get_arguments()
     init_logger(debug)
-    logging.debug("[+] Parameters: source= %s, destination= %s, extennsions= %s, debug= %s", source, destination, extensions, debug)
+    logging.debug("[+] Parameters: source= %s, destination= %s, extennsions= %s, debug= %s", source, destination,
+                  extensions, debug)
     image_organizer = FileOrganizer(source, destination, extensions)
     try:
         logging.debug("[+] Starting")
         image_organizer.start()
-    except Exception:
+    except Exception as err:
+        logging.debug(f"Unexpected {err=}, {type(err)=}")
         logging.error("[-] Something went really wrong")
```

Comment:
Do we really want two logs in here ? debug and error ? would not be better to update the error and add value if needed ?

Reply:
From my undeerstanding logging,error is going to show the error when it happends, but error.debug when debug flag is enabled only. I do not want to show the `err` in "normal" mode, but when I'm debugging I want to know what happened.

Reply:
Changed as:

```
    except Exception as err:
        if debug == logging.DEBUG:
            error_msg = f"Unexpected {err=}, {type(err)=}"
        else:
            error_msg = "[-] Something went really wrong"
        logging.error(error_msg)
```